### ISSUE TITLE
Add a `has-conflicts` badge to PRs automatically

### ISSUE THREAD

**Issue by ashmaroli** [parkr/auto-reply#39]
GitHub status already reports  a list of conflicts in a PR and disables `merge` as a consequence.
It'd be better if JekyllBot could piggyback on that status and automatically add a `has-conflicts` label to such PRs or use Jekyll's existing `pending-resolution` label.

**Comment by pathawks** [comment]
Maybe also add a comment to the PR letting the author/maintainers know that there are new conflicts that need to be resolved?

I'm not sure exactly how this would work though. Would every open PR need to be checked for conflicts each time a commit is added to `master`?

**Comment by DirtyF** [comment]
> We, the maintainers then shout-out to the author if we're not able to resolve the conflicts ourselves

👎 I'd rather minimize a comment that handle all conflicts.

**Comment by ashmaroli** [comment]
> I'd rather minimize a comment that handle all conflicts.

Probably that's how the rest of team feels as well.. :+1:

**Comment by DirtyF** [comment]
According to https://developer.github.com/v3/repos/merging/ @jekyllbot could add a label is a merge returns a `Status: 409 Conflict`

**Comment by parkr** [comment]
Yeah, checking if we get a 409 and adding some kind of `unmergable` label would be helpful feedback for maintainers, especially those using mobile. 📱
